src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -139,15 +141,27 @@ public Integer getTotalUnReviewedDars(AuthUser authUser) {
     /**
      * Filter DataAccessRequestManage objects on user.
      *
-     * @param authUser Filter on what DACs the auth user has access to.
+     * @param user Filter on what DARs the user has access to.
      * @return List of DataAccessRequestManage objects
      */
-    public List<DataAccessRequestManage> describeDataAccessRequestManageV2(AuthUser authUser) {
+    public List<DataAccessRequestManage> describeDataAccessRequestManageV2(User user, Optional<String> roleName) {
+        if (roleName.isPresent()) {
+            if (roleName.get().equalsIgnoreCase(UserRoles.SIGNINGOFFICIAL.getRoleName()) && DarUtil.hasUserRole(user, 7)) {
+                if (Objects.nonNull(user.getInstitutionId())) {
+                    List<DataAccessRequest> dars = dataAccessRequestDAO.findAllDataAccessRequestsForInstitution(user.getInstitutionId());
+                    List<DataAccessRequest> openDars = filterOutCanceledDars(dars);
+                    return createAccessRequestManageV2(openDars);
+                } else {
+                    throw new NotFoundException("Signing Official (user: " + user.getDisplayName() + ", " + user.getDacUserId() + ") "
+                      + "is not associated with an Institution.");
+                }
+            } else {
+                throw new ForbiddenException("User: " + user.getDisplayName() + ", " + user.getDacUserId() + " does not have Signing Official role.");
```

Comment:
nit: remove the user id from the message, we don't need to leak that unnecessarily.

Reply:
If I'm reading this right, I still see the user id in the message.